python_benchmarks/benchmark_layernorm.py

```diff
@@ -0,0 +1,148 @@
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
+from .core import runBenchmark
+import torch
+from .global_params import pytestmark, RTOL, ATOL
+
+
+def layernorm_fwd_fusion(
+    fd: FusionDefinition,
+    dtype: DataType,
+    eps: float = 1e-5,
+) -> None:
+    T0 = fd.define_tensor(
+        shape=[-1, -1], contiguity=[True, True], dtype=dtype, is_cpu=False
+    )
+
+    if dtype is not DataType.Float:
+        T0 = fd.ops.cast(T0, dtype=DataType.Float)
+
+    T2, T3 = fd.ops.var_mean(T0, axes=[1], correction=0, keepdim=False)
+
+    V6 = fd.define_vector([T0.size(0), 1], dtype=DataType.Int)
+    T7 = fd.ops.broadcast_in_dim(T2, shape=V6, broadcast_dims=[0])
+    T11 = fd.ops.broadcast_in_dim(T3, shape=V6, broadcast_dims=[0])
+
+    S12 = fd.define_scalar(eps, dtype=DataType.Double)
+    T13 = fd.ops.add(T7, S12)
+    T14 = fd.ops.rsqrt(T13)
+
+    V17 = fd.define_vector([T0.size(0), T0.size(1)], dtype=DataType.Int)
+    T18 = fd.ops.broadcast_in_dim(T11, shape=V17, broadcast_dims=[0, 1])
+    T19 = fd.ops.sub(T0, T18)
+    T23 = fd.ops.broadcast_in_dim(T14, shape=V17, broadcast_dims=[0, 1])
+    T24 = fd.ops.mul(T19, T23)
+
+    if dtype is not DataType.Float:
+        T24 = fd.ops.cast(T24, dtype=dtype)
+    fd.add_output(T24)
+    fd.add_output(T3)
+    fd.add_output(T14)
+
+
+def layernorm_bwd_fusion(
+    fd: FusionDefinition,
+    dtype: DataType,
```

Comment:
(This is related to my previous comment)

It looks like you are limiting the use of dtype to only some inputs and outputs rather than all of I/Os or any computation. Is this intended so it's in sync with pytorch?